### PR TITLE
Blog literals

### PR DESCRIPTION
**Pull request by sigilante**
Blog post for dev week, covering noun literal syntax for atoms, blobs, and consolidated tuples.

### CODE REVIEW

File: content/blog/literals.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+author = "N E Davis"
+ship = "~lagrev-nocfep"
+image = "https://www.themarginalian.org/wp-content/uploads/2020/01/wilsonbentley_snowflakes0.jpg?fit=1183%2C621&ssl=1"
++++
+
+![](https://www.themarginalian.org/wp-content/uploads/2020/01/wilsonbentley_snowflakes0.jpg?fit=1183%2C621&ssl=1)
+
+#  What Every Hooner Should Know About Literals on Urbit
+
+The Hoon source builder handles conversions between values and applications of auras and molds automatically, and most of the time you won't be surprised.  But sometimes you run into something like this:
```
Comment (tinnus-napbus) [inline content/blog/literals.md:15]:
```suggestion
The Hoon compiler handles conversions between values and applications of auras and molds automatically, and most of the time you won't be surprised.  But sometimes you run into something like this:
```

File: content/blog/literals.md [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+| `@tas` | ASCII text symbol (term) | `%_____` | `%howdy` | Character restrictions; use `++sane`. |
+| `@u` | unsigned integer | — | — | Has no characteristic form. |
+| `@ub` | unsigned binary | `0b____.____` | `0b11.1000` | |
+| `@ud` | unsigned decimal | `___.___` | `1.000.056` | |
+| `@ui` | unsigned decimal | `____` | `0i1000` | |
+| `@uv` | unsigned base32 | `0v_____._____` | `0v1df64.49beg` | |
+| `@uw` | unsigned base64 | `0w_____._____` | `0wbnC.8haTg` | |
+| `@ux` | unsigned hexadecimal | `0x____.____` | `0x5f5.e138` | |
+
+You'll also find some irregular auras in use:  `%lull`, for instance, has a `@uxblob` type.  Nonstandard auras (i.e. those not listed in the table above)
```
Comment (tinnus-napbus) [inline content/blog/literals.md:168]: This line is a bit weird.. maybe typo'd sentence structure?

**Review by tinnus-napbus — APPROVED** [review]
looks good, just a couple of minor things

**Comment by sigilante** [comment]
@matildepark question:  LaTeX isn't rendering inline by default, what do I need to do to fix that?

**Comment by matildepark** [comment]
> @matildepark question: LaTeX isn't rendering inline by default, what do I need to do to fix that?

It should be, looking at the source … let me look in a bit.

**Comment by matildepark** [comment]
Wait. There’s no Math in this post.

**Comment by sigilante** [comment]
$2^0$ after the table

------- Original Message -------

**Pull request merged**